geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/package.scala

```diff
@@ -30,7 +30,9 @@ package object filter {
   implicit val ff: FilterFactory2 = CommonFactoryFinder.getFilterFactory2
 
   object FilterProperties {
-    val GeometryProcessing = SystemProperty("geomesa.geometry.processing", "spatial4j")
+    val GeometryProcessing: SystemProperty = SystemProperty("geomesa.geometry.processing", "spatial4j")
```

Comment:
Did this line change?

Reply:
i just added the type since intellij was complaining about it